safehttp/form_test.go

```diff
@@ -12,43 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package safehttp
+package safehttp_test
```

Comment:
what is the reason for this change?

Reply:
We don't need any parts of the unexported API to perform the tests, so we can use `safehttp_test`. It is good to restrict stuff as much as possible, right? Like don't mark a function as `public` (in other languages than Go) when it doesn't need to be.